api/src/test/java/org/pmiops/workbench/actionaudit/targetproperties/WorkspaceTargetPropertyTest.kt

```diff
@@ -6,20 +6,30 @@ import org.junit.Test
 import org.junit.runner.RunWith
 import org.pmiops.workbench.model.DataAccessLevel
 import org.pmiops.workbench.model.ResearchPurpose
+import org.pmiops.workbench.model.SpecificPopulationEnum
 import org.pmiops.workbench.model.Workspace
 import org.springframework.test.context.junit4.SpringRunner
 
 @RunWith(SpringRunner::class)
 class WorkspaceTargetPropertyTest {
 
+    private val targetPropertyCount = WorkspaceTargetProperty.values().size
```

Comment:
nit: I feel like this can just go inline with where it's being used.

Reply:
👍 It was going to be used multiple times, but I wound up not needing it.